ppr-ui/src/resources/tableHeaders.ts

```diff
@@ -441,56 +441,56 @@ export const registrationTableHeaders: Array<BaseHeaderIF> = [
 export const mhRegistrationTableHeaders: Array<BaseHeaderIF> = [
   {
     class: 'min-column-md',
-    sortable: false,
+    sortable: true,
     text: 'Registration Number',
     value: 'mhrNumber',
     display: true
   },
   {
     class: 'min-column-md',
-    sortable: false,
+    sortable: true,
     text: 'Registration Type',
-    value: 'registrationDescription',
+    value: 'registrationType',
```

Comment:
Was this an api change? 
Or is it being mapped in the UI?

Reply:
API changed to use registrationType for sort in ticket 14336

Reply:
Copy that, thank you.
Does the API still have `registrationDescription` property available then? Otherwise i would have expected the table/values to have broken

Reply:
I'll take a look

Reply:
Just to be safe, I'm going to revert to `registrationDescription` as I see it used in other parts of the API. Going to pass `registrationType` in place of `registrationDescription` to the sort though (as done with `registeringParty` and `ownerNames` in `mhr-api-helper.ts` lines 608-612)

Reply:
Sounds good, thanks for confirming 👍